mlrun/projects/pipelines.py

```diff
@@ -750,6 +754,11 @@ def run(
             runspec = runspec.set_label("job-type", "workflow-runner").set_label(
                 "workflow", workflow_name
             )
+            if workflow_spec.schedule and overwrite:
```

Comment:
I feel like we want to minimize the chance that we deleted the schedule but the new one wasn't created.
If I understand the flow correctly, a POST /submit_job request will be sent to the chief, and only after many validations, a schedule will be created.
Maybe we should move the deletion to the `_submit_run` function.
This way the deletion will also be done by the chief in the same request and we won't need 2 requests for the deletion and creation.
Of course that in order to do so we will need to pass the overwrite flag in the POST request.
Also, this will allow overwriting schedules not only via the CLI.

Reply:
changing